variables.tf

```diff
@@ -150,7 +161,7 @@ variable "object_versioning_enabled" {
 }
 
 variable "archive_days" {
-  description = "Specifies the number of days when the archive rule action takes effect. Only used if 'create_cos_bucket' is true."
+  description = "Specifies the number of days when the archive rule action takes effect. Only used if 'create_cos_bucket' is true. Cannot be set when using var.cross_region_location as archive data is not supported with this feature."
```

Comment:
must be set to `null` when when using var.cross_region_location as archive data is not supported with this feature.